### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mgeplf**
Feedback from @wvangeit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ pip install .
 
 ### NodeStorage
 
-```
->> from libsonata import NodeStorage
+```python
+>> import libsonata
```
Comment (wizmer) [inline README.md]: What is the purpose of the `>>` in the python code block ?

File: README.md [hunk 2]
```diff
@@ -86,29 +88,14 @@ For instance, `{1, 2, 3, 5}` sequence becomes `{[1, 4), [5, 6)}`.
 True
```
Comment (wizmer) [inline README.md]: maybe add:
`>> bool(Selection([]))` here to show how the `bool` is behaving.

File: README.md [hunk 3]
```diff
@@ -119,42 +106,40 @@ Analogous to `NodeStorage`.
...
+# query connectivity (result is Selection object)
+>> selection_to_1 = population.afferent_edges(1)  # all edges with target 1
+>> population.target_nodes(selection_to_1)  # since selection only contains edges targetting 1
+                                            # this will be a numpy array of all 1's
 
-# ...(works for NumPy arrays as well)
->> import numpy as np
->> population.source_nodes(np.asarray([0, 1]))
->> population.target_nodes(np.asarray([0, 1]))
+>> selection_from_2 = population.efferent_edges(2)  # all edges sourced from on 2
```
Comment (wvangeit) [inline README.md]: Mention what '2' is (mention it every time there is a number used)

File: README.md (hunk 2 above)
Comment (mgeplf) [reply README.md]: I'm more leaning to removing the bool completely - it just works like the normal containers

File: README.md (hunk 1 above)
Comment (mgeplf) [reply README.md]: to show the difference between what is input, and what is output

File: README.md (hunk 2 above)
Comment (wizmer) [reply README.md]: Yes, fair enough.

File: README.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ make -j
 
 ## Installing as a Python package
 
-```
+```shell
 git clone git@github.com:BlueBrain/libsonata.git --recursive
```
[the remark this answers is not in the document]
Comment (wizmer) [reply README.md]: I find that one instruction per line is clearer. Also it does not depend on your shell (fish shell does not support `&&` for example)

[the remark this answers is not in the document]
Comment (wvangeit) [reply README.md]: I meant pip install straight from github without cloning

[the remark this answers is not in the document]
Comment (wizmer) [reply README.md]: Ah yes, why not:
`pip install git+ssh://git@github.com/BlueBrain/libsonata.git` works (if you have you ssh setup)
or the https version.

**Comment by mgeplf** [comment]
Thanks for the feedback @wvangeit, if you have more, let us know.

**Pull request merged**